packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_GenericDistribution.res

```diff
@@ -66,6 +66,38 @@ module Helpers = {
       dist1,
     )->runGenericOperation
   }
+  let parseNumber = (args: expressionValue) : Belt.Result.t<float, string> => 
+    switch args {
+    | EvNumber(x) => Ok(x)
+    | _ => Error("Not a number")
+    }
+
+  let parseNumberArray = (ags: array<expressionValue>) : Belt.Result.t<array<float>, string> => E.A.fmap(parseNumber, ags) |> E.A.R.firstErrorOrOpen
+
+  let parseDist = (args: expressionValue): Belt.Result.t<GenericDist_Types.genericDist, string> => 
```

Comment:
Should be `DistributionTypes.genericDist`

Reply:
![image](https://user-images.githubusercontent.com/13807753/163082923-0488b0e9-1c4b-409c-9ca5-908355b18ff3.png)
When I use that, I get a type error. Because Mixture takes a different type of genericDist

Reply:
Also, the rest of this file uses GenericDist_Types.genericDist

Reply:
Maybe we should change this entire file into DistributionTypes.{genericDist, error}? That can be done

Reply:
@Hazelfire Either of those options seems okay to me for this. We could have more rolls of refactors after this PR.

Reply:
Huh? It seems like DistributionTypes isn't used anywhere in the entire project, it's pretty much dead code. Was that intentional? Should I do that refactor now?

Reply:
Oh! Your #232 isn't merged yet. I guess it's a merge race then xD. Whoever merges last has to deal with it. I don't mind either way